nvflare/app_common/workflows/base_model_controller.py

```diff
@@ -113,9 +114,11 @@ def broadcast_model(
             task_name (str, optional): name of the task. Defaults to "train".
             data (FLModel, optional): FLModel to be sent to clients. If no data is given, send empty FLModel.
             targets (List[str], optional): the list of target client names or None (all clients). Defaults to None.
+            min_responses (int, optional): the minimum number of responses expected. If == 0, must receive responses from
+              all clients that the task has been sent to. Defaults to 0.
             timeout (int, optional): time to wait for clients to perform task. Defaults to 0, i.e., never time out.
             wait_time_after_min_received (int, optional): time to wait after
-                minimum number of clients responses has been received. Defaults to 10.
+                minimum number of clients responses has been received. Defaults to 0.
```

Comment:
Our timeout when set to 0 means never timeout VS  wait_time_after_min_received set to 0 means don't wait, is there any conflicts here?
This has nothing to do with this PR though.

Reply:
Good question, there's no conflict here as timeout=0 (never timeout) just means that the task will not end early after 'timeout' seconds regardless if completed.

wait_time_after_min_received means that after min_responses are received, the task will wait x more seconds and then end.